app/controllers/world_controller.rb

```diff
@@ -1,6 +1,6 @@
 class WorldController < ApplicationController
   def index
-    if Features.graphql_feature_enabled?
+    if Features.graphql_feature_enabled? || params.include?(:graphql)
```

Comment:
Now that I've seen this for real, it's got me thinking how about
```suggestion
    if Features.graphql_feature_enabled? && params.include?(:graphql)
```

I just imagined, like, a second ago, what if the feature flag enables the feature to be used at all in the given env and then the query param tells Collections to actually use the GraphQL data source for this request... Would that be pointless?

Reply:
That could be useful, but I wonder if the benefit of rolling out by environment is that we get the benefit of traffic replay where public GET requests are replicated in staging?